scripts/report-bundle-size.js

```diff
@@ -4,18 +4,22 @@
 const path = require('path')
 const childProc = require('child_process')
 
+const bundlesNames = ['main', 'client-optional', 'server-renderer', 'ssr', 'request-processor']
```

Comment:
How about having this list _not_ hardcoded? We can grab the export of webpack/config.js, iterate through the array, and parse the `name` key.

Reply:
@alexvuong sorry, my bad.. pls ignore my advice above. Webpack config is supposed to be read from the context of your project. You were seeing the export of it missing several bundles because the config does the check `entryPointExists`.

Instead of parsing the webpack config, how about you grab the file names of all *.json (except for loadable-stats.json) in the build directory?

(Or if you opt for the simpler hardcoding way, I'll respect that too. I'll leave it up to you)

Reply:
@vmarta thanks for the suggestion. To void grabbing wrong files, I've named those stat json files to have a convention `*-analyzer-stats.json`, and have the `report-bundle-size.js` to check name before sending the data to datadog. I've pushed the change in my latest commint.